packages/website/src/pages/layout/Limit.tsx

```
@@ -1,86 +1,69 @@
-                </label>
-                <div className="form-control">
-                    <Limit id="4" title={title} usage={usageEqualToLimit} isLimitTheGoalToReach limit={100} />
-                </div>
-            </div>
-            <div className="form-group">
-                <label className="form-control-label">
-                    A limit card with a custom limit value and a custom limit title
-                </label>
-                <div className="form-control">
-                    <Limit id="5" title={title} usage={usage} limit={customLimit} limitLabel={customLimitTitle} />
-                </div>
-            </div>
-            <div className="form-group">
-                <label className="form-control-label">A limit card with a modifiable limit value</label>
-                <div className="form-control">
-                    <Limit id="6" title={title} usage={usage} isLimitEditable limit={100} />
-                </div>
-            </div>
-            <div className="form-group">
-                <label className="form-control-label">A limit card without a usage value</label>
-                <div className="form-control">
-                    <Limit id="7" title={title} isLimitEditable limit={100} />
-                </div>
-            </div>
-            <div className="form-group">
-                <label className="form-control-label">A limit card without a limit, but with usage </label>
-                <div className="form-control">
-                    <Limit id="8" title={title} usage={17} isLimitEditable />
-                </div>
-            </div>
-        </div>
-    </PlasmaComponent>
+        componentSourcePath="/limit/Limit.tsx"
+        title="Limit card"
+        section="Layout"
+        description="A limit card displays the limit and usage of a resource. It includes a bar illustrating the usage against the limit."
+        thumbnail="placeholder"
```

this prop defaults to placeholder, no need to specify it

```suggestion
```